### DISCUSSION TITLE
Camunda y otras herramientas para  diagrama bpmn

### DISCUSSION THREAD

**Discussion by xramons** [Launch-X-Latam/MisionFrontEnd#195] [Ideas]
Hola, queria comentar que utilice camunda para hacer el diagrama 

https://modeler.cloud.camunda.io/share/b7bed413-0f17-41b9-aad2-627f8c93e774

Tiene opciones para crear endpoint rest, y simular tokens, me parece muy completo.

Queria consultar si pueden compartir que otras herramientas usaron para bpmn.

Saludos.

**Comment by doguedogue** [comment]
genial, yo he usado Bizagi Modeler (aplicación de escritorio) y draw.io (página) las dos son buenas, la primera es más completa y creo que tiene versión de pago 🚀
